app/assets/stylesheets/_variables.scss

```diff
@@ -0,0 +1 @@
+
```

Comment:
Is this needed? I suppose you could move that colour variable mentioned in another comment into here?